L 10/20/2019 - 00:00:55: SourceMod error session started
L 10/20/2019 - 00:00:55: Info (map "c2m2_fairgrounds") (file "errors_20191020.log")
L 10/20/2019 - 00:00:55: Error log file session closed.
L 10/20/2019 - 00:09:36: SourceMod error session started
L 10/20/2019 - 00:09:36: Info (map "c2m2_fairgrounds") (file "errors_20191020.log")
L 10/20/2019 - 00:09:36: [SM] Exception reported: Entity 11 (11) is invalid
L 10/20/2019 - 00:09:36: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/20/2019 - 00:09:36: [SM] Call stack trace:
L 10/20/2019 - 00:09:36: [SM]   [0] GetEntDataEnt2
L 10/20/2019 - 00:09:36: [SM]   [1] Line 115, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::Event_PlayerTeam
L 10/20/2019 - 00:09:36: [SM] Exception reported: Entity 10 (10) is invalid
L 10/20/2019 - 00:09:36: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/20/2019 - 00:09:36: [SM] Call stack trace:
L 10/20/2019 - 00:09:36: [SM]   [0] GetEntDataEnt2
L 10/20/2019 - 00:09:36: [SM]   [1] Line 115, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::Event_PlayerTeam
L 10/20/2019 - 00:14:08: [SM] Exception reported: Client 7 is not in game
L 10/20/2019 - 00:14:08: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/20/2019 - 00:14:08: [SM] Call stack trace:
L 10/20/2019 - 00:14:08: [SM]   [0] GetClientTeam
L 10/20/2019 - 00:14:08: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/20/2019 - 00:14:30: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/20/2019 - 00:14:30: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/20/2019 - 00:14:30: [SM] Call stack trace:
L 10/20/2019 - 00:14:30: [SM]   [0] SetEntProp
L 10/20/2019 - 00:14:30: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/20/2019 - 00:16:02: Error log file session closed.
L 10/20/2019 - 00:28:12: SourceMod error session started
L 10/20/2019 - 00:28:12: Info (map "c2m4_barns") (file "errors_20191020.log")
L 10/20/2019 - 00:28:12: [SM] Exception reported: Invalid edict (-1 - -1)
L 10/20/2019 - 00:28:12: [SM] Blaming: l4d2_jockey_skeet.smx
L 10/20/2019 - 00:28:12: [SM] Call stack trace:
L 10/20/2019 - 00:28:12: [SM]   [0] GetEdictClassname
L 10/20/2019 - 00:28:12: [SM]   [1] Line 141, plugin/EUplugins/march/l4d2_jockey_skeet.sp::IsShotgun
L 10/20/2019 - 00:28:12: [SM]   [2] Line 73, plugin/EUplugins/march/l4d2_jockey_skeet.sp::OnTakeDamage
L 10/20/2019 - 00:34:37: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/20/2019 - 00:34:37: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/20/2019 - 00:34:37: [SM] Call stack trace:
L 10/20/2019 - 00:34:37: [SM]   [0] SetEntProp
L 10/20/2019 - 00:34:37: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/20/2019 - 00:35:42: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/20/2019 - 00:35:42: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/20/2019 - 00:35:42: [SM] Call stack trace:
L 10/20/2019 - 00:35:42: [SM]   [0] SetEntProp
L 10/20/2019 - 00:35:42: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/20/2019 - 00:39:14: Error log file session closed.
L 10/20/2019 - 00:51:38: SourceMod error session started
L 10/20/2019 - 00:51:38: Info (map "c2m5_concert") (file "errors_20191020.log")
L 10/20/2019 - 00:51:38: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/20/2019 - 00:51:38: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/20/2019 - 00:51:38: [SM] Call stack trace:
L 10/20/2019 - 00:51:38: [SM]   [0] SetEntProp
L 10/20/2019 - 00:51:38: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/20/2019 - 00:52:01: Error log file session closed.
L 10/20/2019 - 00:54:49: SourceMod error session started
L 10/20/2019 - 00:54:49: Info (map "c3m1_plankcountry") (file "errors_20191020.log")
L 10/20/2019 - 00:54:49: [SM] Exception reported: Client index 240 is invalid
L 10/20/2019 - 00:54:49: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/20/2019 - 00:54:49: [SM] Call stack trace:
L 10/20/2019 - 00:54:49: [SM]   [0] IsClientInGame
L 10/20/2019 - 00:54:49: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/20/2019 - 00:54:49: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/20/2019 - 00:54:49: [SM] Exception reported: Client index 240 is invalid
L 10/20/2019 - 00:54:49: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/20/2019 - 00:54:49: [SM] Call stack trace:
L 10/20/2019 - 00:54:49: [SM]   [0] IsClientInGame
L 10/20/2019 - 00:54:49: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/20/2019 - 00:54:49: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/20/2019 - 01:00:06: [SM] Exception reported: Entity 10 (10) is invalid
L 10/20/2019 - 01:00:06: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/20/2019 - 01:00:06: [SM] Call stack trace:
L 10/20/2019 - 01:00:06: [SM]   [0] GetEntDataEnt2
L 10/20/2019 - 01:00:06: [SM]   [1] Line 115, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::Event_PlayerTeam
L 10/20/2019 - 01:04:30: [SM] Exception reported: Client index 220 is invalid
L 10/20/2019 - 01:04:30: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/20/2019 - 01:04:30: [SM] Call stack trace:
L 10/20/2019 - 01:04:30: [SM]   [0] IsClientInGame
L 10/20/2019 - 01:04:30: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/20/2019 - 01:04:30: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/20/2019 - 01:04:30: [SM] Exception reported: Client index 220 is invalid
L 10/20/2019 - 01:04:30: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/20/2019 - 01:04:30: [SM] Call stack trace:
L 10/20/2019 - 01:04:30: [SM]   [0] IsClientInGame
L 10/20/2019 - 01:04:30: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/20/2019 - 01:04:30: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/20/2019 - 01:10:00: [SM] Exception reported: Entity 2 (2) is invalid
L 10/20/2019 - 01:10:00: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/20/2019 - 01:10:00: [SM] Call stack trace:
L 10/20/2019 - 01:10:00: [SM]   [0] GetEntDataEnt2
L 10/20/2019 - 01:10:00: [SM]   [1] Line 89, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/20/2019 - 01:10:01: Error log file session closed.
L 10/20/2019 - 01:18:50: SourceMod error session started
L 10/20/2019 - 01:18:50: Info (map "c3m2_swamp") (file "errors_20191020.log")
L 10/20/2019 - 01:18:50: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/20/2019 - 01:18:50: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/20/2019 - 01:18:50: [SM] Call stack trace:
L 10/20/2019 - 01:18:50: [SM]   [0] SetEntProp
L 10/20/2019 - 01:18:50: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/20/2019 - 01:20:23: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/20/2019 - 01:20:23: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/20/2019 - 01:20:23: [SM] Call stack trace:
L 10/20/2019 - 01:20:23: [SM]   [0] SetEntProp
L 10/20/2019 - 01:20:23: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/20/2019 - 01:20:59: Error log file session closed.
L 10/20/2019 - 01:32:30: SourceMod error session started
L 10/20/2019 - 01:32:30: Info (map "c3m3_shantytown") (file "errors_20191020.log")
L 10/20/2019 - 01:32:30: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/20/2019 - 01:32:30: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/20/2019 - 01:32:30: [SM] Call stack trace:
L 10/20/2019 - 01:32:30: [SM]   [0] SetEntProp
L 10/20/2019 - 01:32:30: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/20/2019 - 01:39:49: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/20/2019 - 01:39:49: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/20/2019 - 01:39:49: [SM] Call stack trace:
L 10/20/2019 - 01:39:49: [SM]   [0] SetEntProp
L 10/20/2019 - 01:39:49: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/20/2019 - 01:40:27: [SM] Exception reported: Client 5 is not in game
L 10/20/2019 - 01:40:27: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/20/2019 - 01:40:27: [SM] Call stack trace:
L 10/20/2019 - 01:40:27: [SM]   [0] GetClientTeam
L 10/20/2019 - 01:40:27: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/20/2019 - 01:40:28: Error log file session closed.
L 10/20/2019 - 01:45:02: SourceMod error session started
L 10/20/2019 - 01:45:02: Info (map "c3m4_plantation") (file "errors_20191020.log")
L 10/20/2019 - 01:45:02: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/20/2019 - 01:45:02: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/20/2019 - 01:45:02: [SM] Call stack trace:
L 10/20/2019 - 01:45:02: [SM]   [0] SetEntProp
L 10/20/2019 - 01:45:02: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/20/2019 - 01:45:44: [SM] Exception reported: Entity 4 (4) is invalid
L 10/20/2019 - 01:45:44: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/20/2019 - 01:45:44: [SM] Call stack trace:
L 10/20/2019 - 01:45:44: [SM]   [0] GetEntDataEnt2
L 10/20/2019 - 01:45:44: [SM]   [1] Line 115, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::Event_PlayerTeam
L 10/20/2019 - 01:48:30: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/20/2019 - 01:48:30: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/20/2019 - 01:48:30: [SM] Call stack trace:
L 10/20/2019 - 01:48:30: [SM]   [0] SetEntProp
L 10/20/2019 - 01:48:30: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/20/2019 - 01:49:45: Error log file session closed.
L 10/20/2019 - 01:49:52: SourceMod error session started
L 10/20/2019 - 01:49:52: Info (map "c1m1_hotel") (file "errors_20191020.log")
L 10/20/2019 - 01:49:52: [SM] Exception reported: Client 7 is not in game
L 10/20/2019 - 01:49:52: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/20/2019 - 01:49:52: [SM] Call stack trace:
L 10/20/2019 - 01:49:52: [SM]   [0] GetClientTeam
L 10/20/2019 - 01:49:52: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/20/2019 - 01:53:17: [SM] Exception reported: Client index 68 is invalid
L 10/20/2019 - 01:53:17: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/20/2019 - 01:53:17: [SM] Call stack trace:
L 10/20/2019 - 01:53:17: [SM]   [0] IsClientInGame
L 10/20/2019 - 01:53:17: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/20/2019 - 01:53:17: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/20/2019 - 01:56:32: [SM] Exception reported: Client 1 is not in game
L 10/20/2019 - 01:56:32: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/20/2019 - 01:56:32: [SM] Call stack trace:
L 10/20/2019 - 01:56:32: [SM]   [0] GetClientTeam
L 10/20/2019 - 01:56:32: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/20/2019 - 02:01:58: [SM] Exception reported: Client index 165 is invalid
L 10/20/2019 - 02:01:58: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/20/2019 - 02:01:58: [SM] Call stack trace:
L 10/20/2019 - 02:01:58: [SM]   [0] IsClientInGame
L 10/20/2019 - 02:01:58: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/20/2019 - 02:01:58: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/20/2019 - 02:01:59: [SM] Exception reported: Client index 165 is invalid
L 10/20/2019 - 02:01:59: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/20/2019 - 02:01:59: [SM] Call stack trace:
L 10/20/2019 - 02:01:59: [SM]   [0] IsClientInGame
L 10/20/2019 - 02:01:59: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/20/2019 - 02:01:59: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/20/2019 - 02:02:00: [SM] Exception reported: Client index 165 is invalid
L 10/20/2019 - 02:02:00: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/20/2019 - 02:02:00: [SM] Call stack trace:
L 10/20/2019 - 02:02:00: [SM]   [0] IsClientInGame
L 10/20/2019 - 02:02:00: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/20/2019 - 02:02:00: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/20/2019 - 02:02:04: [SM] Exception reported: Client index 165 is invalid
L 10/20/2019 - 02:02:04: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/20/2019 - 02:02:04: [SM] Call stack trace:
L 10/20/2019 - 02:02:04: [SM]   [0] IsClientInGame
L 10/20/2019 - 02:02:04: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/20/2019 - 02:02:04: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/20/2019 - 02:02:05: [SM] Exception reported: Client index 165 is invalid
L 10/20/2019 - 02:02:05: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/20/2019 - 02:02:05: [SM] Call stack trace:
L 10/20/2019 - 02:02:05: [SM]   [0] IsClientInGame
L 10/20/2019 - 02:02:05: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/20/2019 - 02:02:05: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/20/2019 - 02:05:52: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/20/2019 - 02:05:52: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/20/2019 - 02:05:52: [SM] Call stack trace:
L 10/20/2019 - 02:05:52: [SM]   [0] SetEntProp
L 10/20/2019 - 02:05:52: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/20/2019 - 02:05:54: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/20/2019 - 02:05:54: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/20/2019 - 02:05:54: [SM] Call stack trace:
L 10/20/2019 - 02:05:54: [SM]   [0] SetEntProp
L 10/20/2019 - 02:05:54: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/20/2019 - 02:06:31: Error log file session closed.
L 10/20/2019 - 02:09:38: SourceMod error session started
L 10/20/2019 - 02:09:38: Info (map "c1m2_streets") (file "errors_20191020.log")
L 10/20/2019 - 02:09:38: [SM] Exception reported: Client 8 is not in game
L 10/20/2019 - 02:09:38: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/20/2019 - 02:09:38: [SM] Call stack trace:
L 10/20/2019 - 02:09:38: [SM]   [0] GetClientTeam
L 10/20/2019 - 02:09:38: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/20/2019 - 02:14:35: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/20/2019 - 02:14:35: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/20/2019 - 02:14:35: [SM] Call stack trace:
L 10/20/2019 - 02:14:35: [SM]   [0] SetEntProp
L 10/20/2019 - 02:14:35: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/20/2019 - 02:20:05: Error log file session closed.
L 10/20/2019 - 02:27:20: SourceMod error session started
L 10/20/2019 - 02:27:20: Info (map "c1m3_mall") (file "errors_20191020.log")
L 10/20/2019 - 02:27:20: [SM] Exception reported: Entity 8 (8) is invalid
L 10/20/2019 - 02:27:20: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/20/2019 - 02:27:20: [SM] Call stack trace:
L 10/20/2019 - 02:27:20: [SM]   [0] GetEntDataEnt2
L 10/20/2019 - 02:27:20: [SM]   [1] Line 115, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::Event_PlayerTeam
L 10/20/2019 - 02:27:20: Error log file session closed.
L 10/20/2019 - 02:29:37: SourceMod error session started
L 10/20/2019 - 02:29:37: Info (map "c1m4_atrium") (file "errors_20191020.log")
L 10/20/2019 - 02:29:37: [SM] Exception reported: Client index 433 is invalid
L 10/20/2019 - 02:29:37: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/20/2019 - 02:29:37: [SM] Call stack trace:
L 10/20/2019 - 02:29:37: [SM]   [0] IsClientInGame
L 10/20/2019 - 02:29:37: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/20/2019 - 02:29:37: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/20/2019 - 02:29:38: [SM] Exception reported: Client index 433 is invalid
L 10/20/2019 - 02:29:38: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/20/2019 - 02:29:38: [SM] Call stack trace:
L 10/20/2019 - 02:29:38: [SM]   [0] IsClientInGame
L 10/20/2019 - 02:29:38: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/20/2019 - 02:29:38: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/20/2019 - 02:32:29: [SM] Exception reported: Entity 10 (10) is invalid
L 10/20/2019 - 02:32:29: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/20/2019 - 02:32:29: [SM] Call stack trace:
L 10/20/2019 - 02:32:29: [SM]   [0] GetEntDataEnt2
L 10/20/2019 - 02:32:29: [SM]   [1] Line 115, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::Event_PlayerTeam
L 10/20/2019 - 02:33:58: [SM] Exception reported: Client index 278 is invalid
L 10/20/2019 - 02:33:58: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/20/2019 - 02:33:58: [SM] Call stack trace:
L 10/20/2019 - 02:33:58: [SM]   [0] IsClientInGame
L 10/20/2019 - 02:33:58: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/20/2019 - 02:33:58: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/20/2019 - 02:35:03: [SM] Exception reported: Entity 4 (4) is invalid
L 10/20/2019 - 02:35:03: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/20/2019 - 02:35:03: [SM] Call stack trace:
L 10/20/2019 - 02:35:03: [SM]   [0] GetEntDataEnt2
L 10/20/2019 - 02:35:03: [SM]   [1] Line 89, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/20/2019 - 02:35:04: Error log file session closed.
L 10/20/2019 - 02:35:07: SourceMod error session started
L 10/20/2019 - 02:35:07: Info (map "c2m1_highway") (file "errors_20191020.log")
L 10/20/2019 - 02:35:07: [SM] Exception reported: Client 6 is not in game
L 10/20/2019 - 02:35:07: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/20/2019 - 02:35:07: [SM] Call stack trace:
L 10/20/2019 - 02:35:07: [SM]   [0] GetClientTeam
L 10/20/2019 - 02:35:07: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/20/2019 - 02:39:03: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/20/2019 - 02:39:03: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/20/2019 - 02:39:03: [SM] Call stack trace:
L 10/20/2019 - 02:39:03: [SM]   [0] SetEntProp
L 10/20/2019 - 02:39:03: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/20/2019 - 02:43:38: SourceMod error session started
L 10/20/2019 - 02:43:38: Info (map "c2m1_highway") (file "errors_20191020.log")
L 10/20/2019 - 02:43:38: [CRASH] Accelerator uploaded crash dump: Crash ID: QROX-476S-D2J7
L 10/20/2019 - 03:24:13: Error log file session closed.
L 10/20/2019 - 03:35:08: SourceMod error session started
L 10/20/2019 - 03:35:08: Info (map "c8m1_apartment") (file "errors_20191020.log")
L 10/20/2019 - 03:35:08: [SM] Exception reported: Entity 3 (3) is invalid
L 10/20/2019 - 03:35:08: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/20/2019 - 03:35:08: [SM] Call stack trace:
L 10/20/2019 - 03:35:08: [SM]   [0] GetEntDataEnt2
L 10/20/2019 - 03:35:08: [SM]   [1] Line 89, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/20/2019 - 03:35:08: [SM] Exception reported: Client 9 is not in game
L 10/20/2019 - 03:35:08: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/20/2019 - 03:35:08: [SM] Call stack trace:
L 10/20/2019 - 03:35:08: [SM]   [0] GetClientTeam
L 10/20/2019 - 03:35:08: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/20/2019 - 03:35:09: Error log file session closed.
L 10/20/2019 - 03:38:28: SourceMod error session started
L 10/20/2019 - 03:38:28: Info (map "c8m2_subway") (file "errors_20191020.log")
L 10/20/2019 - 03:38:28: [SM] Exception reported: Client index 145 is invalid
L 10/20/2019 - 03:38:28: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/20/2019 - 03:38:28: [SM] Call stack trace:
L 10/20/2019 - 03:38:28: [SM]   [0] IsClientInGame
L 10/20/2019 - 03:38:28: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/20/2019 - 03:38:28: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/20/2019 - 03:38:29: [SM] Exception reported: Client index 145 is invalid
L 10/20/2019 - 03:38:29: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/20/2019 - 03:38:29: [SM] Call stack trace:
L 10/20/2019 - 03:38:29: [SM]   [0] IsClientInGame
L 10/20/2019 - 03:38:29: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/20/2019 - 03:38:29: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/20/2019 - 03:47:35: Error log file session closed.
L 10/20/2019 - 03:51:52: SourceMod error session started
L 10/20/2019 - 03:51:52: Info (map "c8m3_sewers") (file "errors_20191020.log")
L 10/20/2019 - 03:51:52: [SM] Exception reported: Client index 214 is invalid
L 10/20/2019 - 03:51:52: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/20/2019 - 03:51:52: [SM] Call stack trace:
L 10/20/2019 - 03:51:52: [SM]   [0] IsClientInGame
L 10/20/2019 - 03:51:52: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/20/2019 - 03:51:52: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/20/2019 - 03:52:05: [SM] Exception reported: Client index 214 is invalid
L 10/20/2019 - 03:52:05: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/20/2019 - 03:52:05: [SM] Call stack trace:
L 10/20/2019 - 03:52:05: [SM]   [0] IsClientInGame
L 10/20/2019 - 03:52:05: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/20/2019 - 03:52:05: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/20/2019 - 03:52:07: [SM] Exception reported: Client index 214 is invalid
L 10/20/2019 - 03:52:07: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/20/2019 - 03:52:07: [SM] Call stack trace:
L 10/20/2019 - 03:52:07: [SM]   [0] IsClientInGame
L 10/20/2019 - 03:52:07: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/20/2019 - 03:52:07: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/20/2019 - 03:52:08: [SM] Exception reported: Client index 214 is invalid
L 10/20/2019 - 03:52:08: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/20/2019 - 03:52:08: [SM] Call stack trace:
L 10/20/2019 - 03:52:08: [SM]   [0] IsClientInGame
L 10/20/2019 - 03:52:08: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/20/2019 - 03:52:08: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/20/2019 - 03:52:08: [SM] Exception reported: Client index 214 is invalid
L 10/20/2019 - 03:52:08: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/20/2019 - 03:52:08: [SM] Call stack trace:
L 10/20/2019 - 03:52:08: [SM]   [0] IsClientInGame
L 10/20/2019 - 03:52:08: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/20/2019 - 03:52:08: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/20/2019 - 03:52:08: [SM] Exception reported: Client index 214 is invalid
L 10/20/2019 - 03:52:08: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/20/2019 - 03:52:08: [SM] Call stack trace:
L 10/20/2019 - 03:52:08: [SM]   [0] IsClientInGame
L 10/20/2019 - 03:52:08: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/20/2019 - 03:52:08: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/20/2019 - 03:52:08: [SM] Exception reported: Client index 214 is invalid
L 10/20/2019 - 03:52:08: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/20/2019 - 03:52:08: [SM] Call stack trace:
L 10/20/2019 - 03:52:08: [SM]   [0] IsClientInGame
L 10/20/2019 - 03:52:08: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/20/2019 - 03:52:08: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/20/2019 - 03:52:09: [SM] Exception reported: Client index 214 is invalid
L 10/20/2019 - 03:52:09: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/20/2019 - 03:52:09: [SM] Call stack trace:
L 10/20/2019 - 03:52:09: [SM]   [0] IsClientInGame
L 10/20/2019 - 03:52:09: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/20/2019 - 03:52:09: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/20/2019 - 03:52:10: [SM] Exception reported: Client index 214 is invalid
L 10/20/2019 - 03:52:10: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/20/2019 - 03:52:10: [SM] Call stack trace:
L 10/20/2019 - 03:52:10: [SM]   [0] IsClientInGame
L 10/20/2019 - 03:52:10: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/20/2019 - 03:52:10: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/20/2019 - 03:55:33: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/20/2019 - 03:55:33: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/20/2019 - 03:55:33: [SM] Call stack trace:
L 10/20/2019 - 03:55:33: [SM]   [0] SetEntProp
L 10/20/2019 - 03:55:33: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/20/2019 - 04:06:19: Error log file session closed.
L 10/20/2019 - 04:13:54: SourceMod error session started
L 10/20/2019 - 04:13:54: Info (map "c8m4_interior") (file "errors_20191020.log")
L 10/20/2019 - 04:13:54: [SM] Exception reported: Entity 5 (5) is invalid
L 10/20/2019 - 04:13:54: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/20/2019 - 04:13:54: [SM] Call stack trace:
L 10/20/2019 - 04:13:54: [SM]   [0] GetEntDataEnt2
L 10/20/2019 - 04:13:54: [SM]   [1] Line 115, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::Event_PlayerTeam
L 10/20/2019 - 04:15:20: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/20/2019 - 04:15:20: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/20/2019 - 04:15:20: [SM] Call stack trace:
L 10/20/2019 - 04:15:20: [SM]   [0] SetEntProp
L 10/20/2019 - 04:15:20: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/20/2019 - 04:15:21: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/20/2019 - 04:15:21: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/20/2019 - 04:15:21: [SM] Call stack trace:
L 10/20/2019 - 04:15:21: [SM]   [0] SetEntProp
L 10/20/2019 - 04:15:21: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/20/2019 - 04:16:13: Error log file session closed.
L 10/20/2019 - 04:17:00: SourceMod error session started
L 10/20/2019 - 04:17:00: Info (map "c8m5_rooftop") (file "errors_20191020.log")
L 10/20/2019 - 04:17:00: [SM] Exception reported: Client 6 is not in game
L 10/20/2019 - 04:17:00: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/20/2019 - 04:17:00: [SM] Call stack trace:
L 10/20/2019 - 04:17:00: [SM]   [0] GetClientTeam
L 10/20/2019 - 04:17:00: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/20/2019 - 04:17:55: [SM] Exception reported: Client 7 is not in game
L 10/20/2019 - 04:17:55: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/20/2019 - 04:17:55: [SM] Call stack trace:
L 10/20/2019 - 04:17:55: [SM]   [0] GetClientTeam
L 10/20/2019 - 04:17:55: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/20/2019 - 04:26:45: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/20/2019 - 04:26:45: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/20/2019 - 04:26:45: [SM] Call stack trace:
L 10/20/2019 - 04:26:45: [SM]   [0] SetEntProp
L 10/20/2019 - 04:26:45: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/20/2019 - 04:29:29: Error log file session closed.
L 10/20/2019 - 04:30:47: SourceMod error session started
L 10/20/2019 - 04:30:47: Info (map "c2m1_highway") (file "errors_20191020.log")
L 10/20/2019 - 04:30:47: [CRASH] Accelerator uploaded crash dump: Crash ID: BTDS-FPSY-2J6K
L 10/20/2019 - 14:54:33: SourceMod error session started
L 10/20/2019 - 14:54:33: Info (map "c2m1_highway") (file "errors_20191020.log")
L 10/20/2019 - 14:54:33: [UPDATER] --- BEGIN ERRORS FROM AUTOMATIC UPDATER ---
L 10/20/2019 - 14:54:33: [UPDATER] Could not download "http://update.sourcemod.net/update/"
L 10/20/2019 - 14:54:33: [UPDATER] Error: Couldn't resolve host 'update.sourcemod.net'
L 10/20/2019 - 14:54:33: [UPDATER] --- END ERRORS FROM AUTOMATIC UPDATER ---
L 10/20/2019 - 15:22:57: SourceMod error session started
L 10/20/2019 - 15:22:57: Info (map "c2m1_highway") (file "errors_20191020.log")
L 10/20/2019 - 15:22:57: [UPDATER] --- BEGIN ERRORS FROM AUTOMATIC UPDATER ---
L 10/20/2019 - 15:22:57: [UPDATER] Parse error in update manifest: Line contained too many invalid tokens
L 10/20/2019 - 15:22:57: [UPDATER] --- END ERRORS FROM AUTOMATIC UPDATER ---
L 10/20/2019 - 19:35:10: Error log file session closed.
L 10/20/2019 - 19:40:36: SourceMod error session started
L 10/20/2019 - 19:40:36: Info (map "c2m1_highway") (file "errors_20191020.log")
L 10/20/2019 - 19:40:36: [SM] Exception reported: Client 10 is not in game
L 10/20/2019 - 19:40:36: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/20/2019 - 19:40:36: [SM] Call stack trace:
L 10/20/2019 - 19:40:36: [SM]   [0] GetClientTeam
L 10/20/2019 - 19:40:36: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/20/2019 - 19:40:36: [SM] Exception reported: Client 11 is not in game
L 10/20/2019 - 19:40:36: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/20/2019 - 19:40:36: [SM] Call stack trace:
L 10/20/2019 - 19:40:36: [SM]   [0] GetClientTeam
L 10/20/2019 - 19:40:36: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/20/2019 - 19:40:36: Error log file session closed.
L 10/20/2019 - 19:42:28: SourceMod error session started
L 10/20/2019 - 19:42:28: Info (map "c2m1_highway") (file "errors_20191020.log")
L 10/20/2019 - 19:42:28: [SM] Exception reported: Entity 6 (6) is invalid
L 10/20/2019 - 19:42:28: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/20/2019 - 19:42:28: [SM] Call stack trace:
L 10/20/2019 - 19:42:28: [SM]   [0] GetEntDataEnt2
L 10/20/2019 - 19:42:28: [SM]   [1] Line 89, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/20/2019 - 19:42:28: [SM] Exception reported: Entity 6 (6) is invalid
L 10/20/2019 - 19:42:28: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/20/2019 - 19:42:28: [SM] Call stack trace:
L 10/20/2019 - 19:42:28: [SM]   [0] GetEntDataEnt2
L 10/20/2019 - 19:42:28: [SM]   [1] Line 115, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::Event_PlayerTeam
L 10/20/2019 - 19:45:10: Error log file session closed.
L 10/20/2019 - 19:58:37: SourceMod error session started
L 10/20/2019 - 19:58:37: Info (map "c2m1_highway") (file "errors_20191020.log")
L 10/20/2019 - 19:58:37: [SM] Exception reported: Client index 233 is invalid
L 10/20/2019 - 19:58:37: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/20/2019 - 19:58:37: [SM] Call stack trace:
L 10/20/2019 - 19:58:37: [SM]   [0] IsClientInGame
L 10/20/2019 - 19:58:37: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/20/2019 - 19:58:37: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/20/2019 - 19:58:51: Error log file session closed.
L 10/20/2019 - 20:03:38: SourceMod error session started
L 10/20/2019 - 20:03:38: Info (map "c2m2_fairgrounds") (file "errors_20191020.log")
L 10/20/2019 - 20:03:38: [SM] Exception reported: Client index 296 is invalid
L 10/20/2019 - 20:03:38: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/20/2019 - 20:03:38: [SM] Call stack trace:
L 10/20/2019 - 20:03:38: [SM]   [0] IsClientInGame
L 10/20/2019 - 20:03:38: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/20/2019 - 20:03:38: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/20/2019 - 20:08:57: [SM] Exception reported: Client index 447 is invalid
L 10/20/2019 - 20:08:57: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/20/2019 - 20:08:57: [SM] Call stack trace:
L 10/20/2019 - 20:08:57: [SM]   [0] IsClientInGame
L 10/20/2019 - 20:08:57: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/20/2019 - 20:08:57: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/20/2019 - 20:08:58: [SM] Exception reported: Client index 447 is invalid
L 10/20/2019 - 20:08:58: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/20/2019 - 20:08:58: [SM] Call stack trace:
L 10/20/2019 - 20:08:58: [SM]   [0] IsClientInGame
L 10/20/2019 - 20:08:58: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/20/2019 - 20:08:58: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/20/2019 - 20:10:38: [SM] Exception reported: Entity 5 (5) is invalid
L 10/20/2019 - 20:10:38: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/20/2019 - 20:10:38: [SM] Call stack trace:
L 10/20/2019 - 20:10:38: [SM]   [0] GetEntDataEnt2
L 10/20/2019 - 20:10:38: [SM]   [1] Line 89, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/20/2019 - 20:10:39: Error log file session closed.
L 10/20/2019 - 20:14:11: SourceMod error session started
L 10/20/2019 - 20:14:11: Info (map "c2m3_coaster") (file "errors_20191020.log")
L 10/20/2019 - 20:14:11: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/20/2019 - 20:14:11: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/20/2019 - 20:14:11: [SM] Call stack trace:
L 10/20/2019 - 20:14:11: [SM]   [0] SetEntProp
L 10/20/2019 - 20:14:11: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/20/2019 - 20:20:35: [SM] Exception reported: Client index 209 is invalid
L 10/20/2019 - 20:20:35: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/20/2019 - 20:20:35: [SM] Call stack trace:
L 10/20/2019 - 20:20:35: [SM]   [0] IsClientInGame
L 10/20/2019 - 20:20:35: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/20/2019 - 20:20:35: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/20/2019 - 20:21:13: Error log file session closed.
L 10/20/2019 - 20:24:50: SourceMod error session started
L 10/20/2019 - 20:24:50: Info (map "c2m4_barns") (file "errors_20191020.log")
L 10/20/2019 - 20:24:50: [SM] Exception reported: Client index 437 is invalid
L 10/20/2019 - 20:24:50: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/20/2019 - 20:24:50: [SM] Call stack trace:
L 10/20/2019 - 20:24:50: [SM]   [0] IsClientInGame
L 10/20/2019 - 20:24:50: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/20/2019 - 20:24:50: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/20/2019 - 20:27:17: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/20/2019 - 20:27:17: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/20/2019 - 20:27:17: [SM] Call stack trace:
L 10/20/2019 - 20:27:17: [SM]   [0] SetEntProp
L 10/20/2019 - 20:27:17: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/20/2019 - 20:32:19: [SM] Exception reported: Client index 327 is invalid
L 10/20/2019 - 20:32:19: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/20/2019 - 20:32:19: [SM] Call stack trace:
L 10/20/2019 - 20:32:19: [SM]   [0] IsClientInGame
L 10/20/2019 - 20:32:19: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/20/2019 - 20:32:19: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/20/2019 - 20:32:22: [SM] Exception reported: Client index 327 is invalid
L 10/20/2019 - 20:32:22: [SM] Blaming: l4d2_getup_slide_fix.smx
L 10/20/2019 - 20:32:22: [SM] Call stack trace:
L 10/20/2019 - 20:32:22: [SM]   [0] IsClientInGame
L 10/20/2019 - 20:32:22: [SM]   [1] Line 39, C:\Users\Sir\Documents\THE MAP\New Scripting\include\l4d2util_infected.inc::IsInfected
L 10/20/2019 - 20:32:22: [SM]   [2] Line 135, C:\Users\Sir\Documents\THE MAP\New Scripting\l4d2_getup_slide_fix.sp::L4D2_OnStagger
L 10/20/2019 - 20:33:03: [SM] Exception reported: Client 6 is not in game
L 10/20/2019 - 20:33:03: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/20/2019 - 20:33:03: [SM] Call stack trace:
L 10/20/2019 - 20:33:03: [SM]   [0] GetClientTeam
L 10/20/2019 - 20:33:03: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/20/2019 - 20:34:38: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/20/2019 - 20:34:38: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/20/2019 - 20:34:38: [SM] Call stack trace:
L 10/20/2019 - 20:34:38: [SM]   [0] SetEntProp
L 10/20/2019 - 20:34:38: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/20/2019 - 20:34:54: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/20/2019 - 20:34:54: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/20/2019 - 20:34:54: [SM] Call stack trace:
L 10/20/2019 - 20:34:54: [SM]   [0] SetEntProp
L 10/20/2019 - 20:34:54: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/20/2019 - 20:39:23: Error log file session closed.
L 10/20/2019 - 20:41:45: SourceMod error session started
L 10/20/2019 - 20:41:45: Info (map "c2m5_concert") (file "errors_20191020.log")
L 10/20/2019 - 20:41:45: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/20/2019 - 20:41:45: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/20/2019 - 20:41:45: [SM] Call stack trace:
L 10/20/2019 - 20:41:45: [SM]   [0] SetEntProp
L 10/20/2019 - 20:41:45: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/20/2019 - 20:47:17: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/20/2019 - 20:47:17: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/20/2019 - 20:47:17: [SM] Call stack trace:
L 10/20/2019 - 20:47:17: [SM]   [0] SetEntProp
L 10/20/2019 - 20:47:17: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/20/2019 - 20:48:37: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/20/2019 - 20:48:37: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/20/2019 - 20:48:37: [SM] Call stack trace:
L 10/20/2019 - 20:48:37: [SM]   [0] SetEntProp
L 10/20/2019 - 20:48:37: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/20/2019 - 20:55:43: [SM] Exception reported: Client 7 is not in game
L 10/20/2019 - 20:55:43: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/20/2019 - 20:55:43: [SM] Call stack trace:
L 10/20/2019 - 20:55:43: [SM]   [0] GetClientTeam
L 10/20/2019 - 20:55:43: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/20/2019 - 20:57:09: [SM] Exception reported: Client 9 is not in game
L 10/20/2019 - 20:57:09: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/20/2019 - 20:57:09: [SM] Call stack trace:
L 10/20/2019 - 20:57:09: [SM]   [0] GetClientTeam
L 10/20/2019 - 20:57:09: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/20/2019 - 21:02:26: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/20/2019 - 21:02:26: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/20/2019 - 21:02:26: [SM] Call stack trace:
L 10/20/2019 - 21:02:26: [SM]   [0] SetEntProp
L 10/20/2019 - 21:02:26: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/20/2019 - 21:03:26: [SM] Exception reported: Entity 1 (1) is invalid
L 10/20/2019 - 21:03:26: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/20/2019 - 21:03:26: [SM] Call stack trace:
L 10/20/2019 - 21:03:26: [SM]   [0] GetEntDataEnt2
L 10/20/2019 - 21:03:26: [SM]   [1] Line 89, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/20/2019 - 21:03:26: [SM] Exception reported: Entity 7 (7) is invalid
L 10/20/2019 - 21:03:26: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/20/2019 - 21:03:26: [SM] Call stack trace:
L 10/20/2019 - 21:03:26: [SM]   [0] GetEntDataEnt2
L 10/20/2019 - 21:03:26: [SM]   [1] Line 89, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/20/2019 - 21:03:28: Error log file session closed.
L 10/20/2019 - 21:03:31: SourceMod error session started
L 10/20/2019 - 21:03:31: Info (map "c6m1_riverbank") (file "errors_20191020.log")
L 10/20/2019 - 21:03:31: [SM] Exception reported: Client 8 is not in game
L 10/20/2019 - 21:03:31: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/20/2019 - 21:03:31: [SM] Call stack trace:
L 10/20/2019 - 21:03:31: [SM]   [0] GetClientTeam
L 10/20/2019 - 21:03:31: [SM]   [1] Line 79, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/20/2019 - 21:04:28: Error log file session closed.
L 10/20/2019 - 21:26:38: SourceMod error session started
L 10/20/2019 - 21:26:38: Info (map "c10m2_drainage") (file "errors_20191020.log")
L 10/20/2019 - 21:26:38: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/20/2019 - 21:26:38: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/20/2019 - 21:26:38: [SM] Call stack trace:
L 10/20/2019 - 21:26:38: [SM]   [0] SetEntProp
L 10/20/2019 - 21:26:38: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/20/2019 - 21:36:29: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/20/2019 - 21:36:29: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/20/2019 - 21:36:29: [SM] Call stack trace:
L 10/20/2019 - 21:36:29: [SM]   [0] SetEntProp
L 10/20/2019 - 21:36:29: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/20/2019 - 21:36:33: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/20/2019 - 21:36:33: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/20/2019 - 21:36:33: [SM] Call stack trace:
L 10/20/2019 - 21:36:33: [SM]   [0] SetEntProp
L 10/20/2019 - 21:36:33: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/20/2019 - 21:36:36: [SM] Exception reported: Entity -1 (-1) is invalid
L 10/20/2019 - 21:36:36: [SM] Blaming: l4d2_melee_shenanigans.smx
L 10/20/2019 - 21:36:36: [SM] Call stack trace:
L 10/20/2019 - 21:36:36: [SM]   [0] SetEntProp
L 10/20/2019 - 21:36:36: [SM]   [1] Line 142, crash/l4d2_melee_shenanigans.sp::TimedSwap
L 10/20/2019 - 21:37:25: Error log file session closed.
L 10/20/2019 - 21:43:07: SourceMod error session started
L 10/20/2019 - 21:43:07: Info (map "c10m3_ranchhouse") (file "errors_20191020.log")
L 10/20/2019 - 21:43:07: [SM] Exception reported: Entity 1 (1) is invalid
L 10/20/2019 - 21:43:07: [SM] Blaming: l4d2_charger_getup_fix.smx
L 10/20/2019 - 21:43:07: [SM] Call stack trace:
L 10/20/2019 - 21:43:07: [SM]   [0] GetEntDataEnt2
L 10/20/2019 - 21:43:07: [SM]   [1] Line 89, C:\Users\Trevor_2\Desktop\scripting\l4d2_charger_getup_fix.sp::OnClientDisconnect
L 10/20/2019 - 22:03:41: SourceMod error session started
L 10/20/2019 - 22:03:41: Info (map "c2m1_highway") (file "errors_20191020.log")
L 10/20/2019 - 22:03:41: [UPDATER] --- BEGIN ERRORS FROM AUTOMATIC UPDATER ---
L 10/20/2019 - 22:03:41: [UPDATER] Could not download "http://update.sourcemod.net/update/"
L 10/20/2019 - 22:03:41: [UPDATER] Error: couldn't connect to host
L 10/20/2019 - 22:03:41: [UPDATER] --- END ERRORS FROM AUTOMATIC UPDATER ---
L 10/20/2019 - 22:05:57: [CRASH] Accelerator failed to upload crash dump: couldn't connect to host (7)
L 10/20/2019 - 23:13:17: Error log file session closed.
